src/main/java/io/cdap/plugin/gcp/bigquery/sink/AbstractBigQuerySinkConfig.java

```diff
@@ -70,6 +72,11 @@
   @Description("Type of write operation to perform. This can be set to Insert, Update or Upsert.")
   protected String operation;
 
+  @Name(NAME_WRITE_DISPOSITION)
+  @Macro
+  @Description("Type of write disposition to use. This can be set to Empty, Truncate, or Append.")
```

Comment:
this description should match the one given in the documentation

Reply:
Fixed!